src/util/job-name-validation.tsx

```diff
@@ -64,3 +65,63 @@ export function NameError(name: string, trans: TranslationBundle): string {
     'Name must contain only letters, numbers, spaces, periods, hyphens, and underscores'
   );
 }
+
+export function MaxRetryAttemptsError(maxRetryAttempts: string, trans: TranslationBundle): string {
+  // Check for blank
+  if (maxRetryAttempts === '') {
+    return trans.__('You must specify max retry attempts');
+  }
+
+  if (!validIntegerRregex.test(maxRetryAttempts)) {
+    return trans.__(
+      'Max retry attempts must be an integer'
```

Comment:
```suggestion
      'Must be an integer between %1 and %2', min, max
```

Explain the acceptable range in the error